src/utils.ts

```diff
@@ -1,44 +1,62 @@
-import { BlockchainCollectionThirdParty, parseUrn } from '@dcl/urn-resolver'
+import { BlockchainCollectionThirdParty, BlockchainCollectionThirdPartyItem, parseUrn } from '@dcl/urn-resolver'
 import { L1_NETWORKS, L2_NETWORKS } from './types'
 
 type URNsByNetwork = {
   ethereum: { urn: string; type: string }[]
   matic: { urn: string; type: string }[]
+  ethereumThirdParty: { urn: string; type: string }[]
+  maticThirdParty: { urn: string; type: string }[]
 }
 
 const ITEM_TYPES_TO_SPLIT = [
   'blockchain-collection-v1-asset',
   'blockchain-collection-v1-item',
   'blockchain-collection-v2-asset',
   'blockchain-collection-v2-item',
-  'blockchain-collection-linked-wearables-item'
+  'blockchain-collection-third-party',
+  'blockchain-collection-third-party-item'
 ]
 
-export async function splitItemsURNsByNetwork(urnsToSplit: string[]): Promise<URNsByNetwork> {
+export async function splitItemsURNsByTypeAndNetwork(urnsToSplit: string[]): Promise<URNsByNetwork> {
   const ethereum: { urn: string; type: string }[] = []
   const matic: { urn: string; type: string }[] = []
+  const ethereumThirdParty: { urn: string; type: string }[] = []
+  const maticThirdParty: { urn: string; type: string }[] = []
+
   for (const urn of urnsToSplit) {
+    const asset = await parseUrn(urn)
+    if (!asset || !('network' in asset) || !ITEM_TYPES_TO_SPLIT.includes(asset.type)) {
+      continue
+    }
+
     // check if it is a L1 or L2 asset
     // 'ethereum' is included since L1 Mainnet assets include it instead of 'mainnet'
-    if (
-      ![...L1_NETWORKS, 'ethereum'].some((network) => urn.includes(network)) &&
-      !L2_NETWORKS.some((network) => urn.includes(network))
-    ) {
+    if (![...L1_NETWORKS, 'ethereum'].includes(asset.network) && !L2_NETWORKS.includes(asset.network)) {
       continue
     }
 
-    const parsed = await parseUrn(urn)
-    if (parsed && 'network' in parsed && ITEM_TYPES_TO_SPLIT.includes(parsed.type)) {
-      if (L1_NETWORKS.includes(parsed.network)) {
-        ethereum.push({ urn, type: parsed.type })
-      } else if (L2_NETWORKS.includes(parsed.network)) {
-        matic.push({ urn, type: parsed.type })
+    if (L1_NETWORKS.includes(asset.network)) {
+      ethereum.push({ urn: asset.uri.toString(), type: asset.type })
```

Comment:
Is it okay to omit 'ethereum' in the list of `L1_NETWORKS`?

Reply:
Good catch! Added.